modules/creating-custom-live-rhcos-iso.adoc

```diff
@@ -18,71 +18,47 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
 
 . Download the `coreos-installer` binary from the `coreos-installer` image link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[mirror] page.
 
-. Download the latest live {op-system} ISO from link:https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/latest/rhcos-live.x86_64.iso[mirror.openshift.com].
+. Download the latest live {op-system} ISO from link:https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/{product-version}/latest/[mirror.openshift.com].
 
 . Create the `embedded.yaml` file that the `butane` utility uses to create the Ignition file:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
-variant: rhcos
-version: 0.1.0
+variant: openshift
+version: {product-version}.0
+metadata:
+  name: sshd
+  labels:
+    machineconfiguration.openshift.io/role: worker
 passwd:
   users:
-    - name: root
+    - name: core <1>
       ssh_authorized_keys:
         - '<ssh_key>'
-      password_hash: '$2b$05$aAd9TkvWnvUmeDQEA9vkiOoCqCZgd6cWn0xXwe.ii/TqeCxUC0Z0G' <1>
-storage:
-  files:
-    - path: /etc/ssh/sshd_config
-      contents:
-        local: ./live-sshd_config
-      mode: 0600
-      overwrite: true
 ----
-<1> `yescrypt` passwords are not supported in {op-system}. Use `bcrypt` instead when you create the `password_hash`.
+<1> The `core` user has sudo privileges.
 
-. Create the `files-dir/` folder and copy the default `sshd_config` file into the folder.
-
-. Update the following fields in the `files-dir/sshd_config` file:
-+
-[source,bash]
-----
-[...]
-PermitRootLogin yes <1>
-[...]
-PasswordAuthentication yes <2>
-----
-<1> Allows access to the server as a root user.
-<2> Allows access to the server by using a password.
-
-+
-[IMPORTANT]
-====
-The modified `sshd_config` file overrides the default `sshd_config` file.
-====
-
-. Pass the YAML file and the folder to the `butane` utility:
+. Run the `butane` utility to create the Ignition file using the following command:
 +
 [source,terminal]
 ----
-$ butane -p embedded.yaml -d files-dir/ > embedded.ign
+$ butane -pr embedded-rhcos.yaml -o embedded-rhcos.ign
```

Comment:
Sorry, please remove `-rhcos` from the file names. I added this string while testing. The name of files `embedded.yaml` and `embedded.ign` should match the previous and later mentions.

Same for `-p (pretty format)`. The command should be
`$ butane -r embedded.yaml -o embedded.ign`

Reply:
updated, thanks for the correction!